packages/apps/plugins/plugin-thread/src/ThreadPlugin.tsx

```diff
@@ -138,13 +138,13 @@ export const ThreadPlugin = (): PluginDefinition<ThreadPluginProvides> => {
               const location = navigationPlugin?.provides.location;
               const activeNode = location?.active ? graph?.findNode(location.active) : undefined;
               const active = activeNode?.data;
-              const space = isDocument(active) && getSpaceForObject(active);
+              const space = active && getSpaceForObject(active);
```

Comment:
```suggestion
              const space = isTypedObject(active) && getSpaceForObject(active);
```
what happens if you pass a non-echo object to `getSpaceForObject`?

Reply:
i think it just returns undefined. adding a test.